cloudflare/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM debian:jessie
```

Comment:
This is probably just an "I'd do it differently criticism," but burying the Dockerfile an extra directory down makes it feel secondary to the point of the repo.